Core/src/main/java/com/rakuten/tech/mobile/perf/core/EnvironmentInfo.java

```diff
@@ -9,7 +9,7 @@
 import java.util.Observable;
 import java.util.Observer;
 
-class EnvironmentInfo implements Observer {
+public class EnvironmentInfo implements Observer {
```

Comment:
I dislike making `EnvironmentInfo` public, it's not intended for use by SDK consumers.
Instead of exposing the class we could:
* move the internal classes in `Runtime` to `com.rakuten.tech.mobile.perf.core` package & use `@hide` to hide it from javadoc -> no additional exposure
* extract the country code lookup into a function and make that public -> limited exposure

- [ ] reduce visibility of `EnvironmentInfo`

Reply:
I extracted this to a Util class. It seems to be a much simpler solution than what I did previously.